build-farm/platform-specific-configurations/linux.sh

```diff
@@ -285,6 +288,16 @@ if [ "${ARCHITECTURE}" == "riscv64" ] && [ "${NATIVE_API_ARCH}" != "riscv64" ];
   if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then
     # shellcheck disable=SC2001
     CONFIGURE_ARGS_FOR_ANY_PLATFORM=$(echo "$CONFIGURE_ARGS_FOR_ANY_PLATFORM" | sed "s,with-openssl=[^ ]*,with-openssl=${RISCV_SYSROOT}/usr,g")
+    if ! which qemu-riscv64-static; then
```

Comment:
I assume the jenkins user will see that executable in its PATH

Reply:
Not sure what your question is here. I put that line in to check if there was already a suitable executable installed on the machine somewhere, in which case that one can be used instead of the downloaded one. A likely precursor to adding something into the playbooks later as part of the default setup.